app/src/main/res/layout/dialog_custom.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"

```

Comment:
WARNING: [UnusedResources] The resource `R.layout.dialog_custom` appears to be unused
 Unused resources make applications larger and slow down builds.